src/com/owncloud/android/lib/resources/activities/GetRemoteActivitiesOperation.java

```diff
@@ -59,7 +59,7 @@
     private static final String TAG = GetRemoteActivitiesOperation.class.getSimpleName();
 
     // OCS Routes
-    private static final String OCS_ROUTE = "/apps/activity/api/v2/activity";
+    private static final String OCS_ROUTE = "/ocs/v2.php/apps/activity/api/v2/activity";
```

Comment:
I think this is wrong, @mario. The correct URL is without the `/ocs/v2.php`. Or is there an OCS API endpoint for Activity, @nickvergessen ? I couldn't find one in the activity code after a quick look.